Tan_ShinYi/2. OOP/human.rb

```diff
@@ -0,0 +1,23 @@
+class Human
+  attr_accessor :strength, :intelligence, :health, :stealth
+  def initialize
+    @strength = 3
+    @stealth = 3
+    @intelligence = 3
+    @health = 100
+  end
+
+  def attack victim
+    if victim.class.ancestors.any?{ |i| i==Human}
```

Comment:
another option here would be to just use include
if victim.class.ancestors.include?(Human)